src/sentry/tsdb/base.py

```diff
@@ -413,16 +415,17 @@ def delete(
     def get_range(
         self,
         model: TSDBModel,
-        keys: list[int],
+        keys: Sequence[TSDBKey],
```

Comment:
anything that was triggering errors on keys got converted over to use `TSDBKey` -- it seems the tsdb store is generic on either having ints or strings as keys